src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -396,4 +407,4 @@ Authorization rules consists of:
 - **authorization provider** (`provider`): which mechanism is used to apply the authorization rule (API Key, IAM, Amazon Cognito user pool, OIDC)
 - **authorized operations** (`operations`): which operations are allowed for the given strategy and provider. If not specified, `create`, `read`, `update`, and `delete` operations are allowed.
 
-**API Keys** are best used for public APIs (or parts of your schema which you wish to be public) or prototyping, and you must specify the expiration time before deploying. **IAM** authorization uses [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) to make request with policies attached to Roles. OIDC tokens provided by **Amazon Cognito user pool** or **3rd party OpenID Connect** providers can also be used for authorization, and simply enabling this provides a simple access control requiring users to authenticate to be granted top level access to API actions.
\ No newline at end of file
+**API Keys** are best used for public APIs (or parts of your schema which you wish to be public) or prototyping, and you must specify the expiration time before deploying. **IAM** authorization uses [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) to make request with policies attached to Roles. OIDC tokens provided by **Amazon Cognito user pool** or **3rd party OpenID Connect** providers can also be used for authorization, and simply enabling this provides a simple access control requiring users to authenticate to be granted top level access to API actions.
```

Comment:
2 nits:
1. Should "User Pool" be capitalized since it is the name of an AWS resource?
2. "simple" is twice in the same sentence, one can probably be eliminated: "simply enabling this provides a simple access control" -> "simply enabling this provides access control"

Reply:
user pool is officially lower case: https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html

addressed the other nit.